app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ContactsController < ApplicationController
+
+  def new
+    @contact = Contact.new
+  end
+
+  def create
+    @contact = Contact.new(params[:contact])
+    @contact.request = request
+    if @contact.deliver
+      flash.now[:notice] = 'Thank you for your message!'
```

Comment:
shouldn't this respond with something?

perhaps at least `head :no_content` or something